evmcc/vendor/modules.txt

```diff
@@ -0,0 +1,337 @@
+# github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
```

Comment:
go mod I don't think is as good as dep at only pulling in the files needed. it pulls in entire projects which means we probably get more dependencies than before.

Reply:
makes sense for non-vendoring where a whole project is the smallest case, and no trimming is necessary because it's not stored in the repo, but is stored locally.